measures/ApplyUpgrade/measure.rb

```diff
@@ -560,9 +568,15 @@ def set_existing_system_as_heat_pump_backup(runner, measures, hpxml_bldg)
 
             # Likely only fuel-fired furnace as integrated backup
             if heat_pump_backup_fuel != HPXML::FuelTypeElectricity
-              measures['ResStockArgumentsPostHPXML'][0]['heat_pump_backup_type'] = heat_pump_backup_type
-              measures['ResStockArgumentsPostHPXML'][0]['heat_pump_backup_fuel'] = heat_pump_backup_fuel
-              measures['ResStockArgumentsPostHPXML'][0]['heat_pump_backup_heating_efficiency'] = heat_pump_backup_heating_efficiency
+              hvac_heat_pump_backup = 'Integrated'
+              hvac_heat_pump_backup += ", #{heat_pump_backup_fuel.capitalize}"
+              if heat_pump_backup_fuel == HPXML::FuelTypeElectricity
+                hvac_heat_pump_backup += ', 100% Efficiency'
+              else
+                hvac_heat_pump_backup += ", #{hvac_heating_system.split(',').map(&:strip)[-1]}"
+              end
+              measures['BuildResidentialHPXML'][0]['hvac_heat_pump_backup'] = hvac_heat_pump_backup
```

Comment:
We may not even need to capture the fuel type, efficiency, etc. Just choose an "Integrated" option, and then fuel type, efficiency, etc. will be reset downstream.